source/slang/slang-check-decl.cpp

```diff
@@ -4090,8 +4129,9 @@ namespace Slang
             }
         }
 
+        bool isDefaultInitializableType = requiredMemberDeclRef.getParent() == getASTBuilder()->getDefaultInitializableTypeInterfaceDecl();
         bool isInWrapperType = isWrapperTypeDecl(context->parentDecl);
-        if (!isInWrapperType)
+        if (!isInWrapperType && !isDefaultInitializableType)
```

Comment:
Can we have a test case for the wrapper-type specifically for this change?
I am not familiar with the wrapper-type and I am not sure if this change will cause any trouble.

Reply:
> // Used if this type declaration is a wrapper, i.e. struct FooWrapper:IFoo = Foo;

^ what we define a wrapperType as in the code

I can add a test case